src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -2016,50 +2017,102 @@ internal override UseSiteInfo<AssemblySymbol> GetUseSiteInfo()
             if (!_lazyCachedUseSiteInfo.IsInitialized)
             {
                 AssemblySymbol primaryDependency = PrimaryDependency;
-                _lazyCachedUseSiteInfo.Initialize(primaryDependency, new UseSiteInfo<AssemblySymbol>(primaryDependency).AdjustDiagnosticInfo(GetUseSiteDiagnosticImpl()));
```

Comment:
>_lazyCachedUseSiteInfo.Initialize(primaryDependency, new UseSiteInfo<AssemblySymbol>(primaryDependency).AdjustDiagnosticInfo(GetUseSiteDiagnosticImpl())); [](http://example.com/codeflow?start=16&length=154)

I am not sure why this has to change. We do not intent to change anything around dependencies tracking. We only can produce different DiagnosticInfo.